api/src/main/resources/application.yaml

```diff
@@ -1,19 +1,37 @@
 server:
-  port: 4013
+    port: 4013
 
 management:
-  endpoint:
-    health:
-      show-details: "always"
-  endpoints:
-    web:
-      base-path: /
-      exposure:
-        include: health, info, prometheus
+    endpoint:
```

Comment:
I'd prefer to stick to 2 spaces if that's okay. We should update the .editorconfig accordingly